components/ShopBuilder/Layouts/BasicLayout.js

```diff
@@ -1,120 +1,37 @@
-//         { id: "3", content: "D", height: "100px" },
-//         { id: "4", content: "E", height: "100px" },
-//         { id: "5", content: "F", height: "100px" }
-//       ]
-//     }
-// ]
+export const ProductExample1 = [
+  { x: 0, y: 0, w: 1, h: 2, content: "a" },
+  { x: 1, y: 0, w: 3, h: 2, content: "b" },
+  { x: 4, y: 0, w: 1, h: 2, content: "c" }
+];
 
-export const BasicLayout = [
-  //This is where the page columns are held which is the layout of the page
-  {
-    id: "MONSTER_IMAGE_1",
-    items: [],
-    order: 0,
-    containerHeight: "750px"
-  },
+export const contentExample = [
   {
-    id: "PRODUCTS_2",
-    items: [
-      //the Id should consist of where it is in the columns array for the
-      //first number and then the index of the item when it was created
-      //for the second number for example 30 is 3 from index 3 in columns and
-      // 0 because it was created at the 0th index
-      { content: "A", height: "100px", width: "150px" },
-      { content: "B", height: "100px", width: "150px" },
-      { content: "C", height: "100px", width: "150px" },
-      { content: "D", height: "100px", width: "150px" },
-      { content: "E", height: "100px", width: "150px" },
-      { content: "F", height: "100px", width: "150px" }
-    ],
-    order: 3,
-    rowLimit: 3
+    content: (
+      <form>
+        <input />
+      </form>
+    )
   },
-  {
-    id: "DOUBLE_SIZE_1",
-    items: [
-      { id: "banner1", content: "Double Size", height: "100px", order: 2 },
-      {
-        id: "halfImage1",
-        content: "half width image",
-        height: "100px",
-        width: "50%",
-        order: 1
-      },
-      {
-        id: "halfImage2",
-        content: "half width image",
-        height: "100px",
-        width: "50%",
-        order: 1
-      }
-    ],
-    order: 1,
-    containerHeight: "150px"
-  },
-  {
-    id: "REGULAR_BANNER_1",
-    items: [],
-    order: 2,
-    containerHeight: "75px"
-  },
-  {
-    id: "PRODUCTS_1",
-    items: [
-      //the Id should consist of where it is in the columns array for the
-      //first number and then the index of the drop area it was created in
-      //for the second number and the final number is it's index
-      { id: "3-0-0", content: "A", height: "100px", width: "150px" },
-      { id: "3-0-1", content: "B", height: "100px", width: "150px" },
-      { id: "3-0-2", content: "C", height: "100px", width: "150px" },
-      { id: "3-1-3", content: "D", height: "100px", width: "150px" },
-      { id: "3-1-4", content: "E", height: "100px", width: "150px" },
-      { id: "3-1-5", content: "F", height: "100px", width: "150px" }
-    ],
-    order: 3,
-    rowLimit: 4
-  }
+  { content: "b" },
+  { content: "c" },
+  { content: "d" }
+];
+
+export const ProductExample3 = [
+  { x: 0, y: 0, w: 1, h: 2, content: "g" },
+  { x: 1, y: 0, w: 3, h: 2, content: "h" },
+  { x: 4, y: 0, w: 1, h: 2, content: "i" }
+];
+
+export const BlankLayout = [
```

Comment:
Tge templating idea has been a great idea overall. We can help pur users express their creative freedom with a blank canvus while at the same time also giving thos who are unsure what direction or who to build a webiste the ablity to select some predefinded templates to help them on their way.